tests/test_models.py

```diff
@@ -14,3 +16,38 @@ def test_package_manager_get_by_name(name, expected, app, db, auth_env):
         assert expected == pkg_manager
     else:
         assert expected == pkg_manager.name
+
+
+class TestRequest:
```

Comment:
I think putting all tests from the same class inside a single test class helps the organization, let me know what you think about it.

Reply:
Great idea! I would just move it to another commit. Not because it's too big, but because it has a different topic.

Reply:
Hm, seems reasonable but: 
1) we already have a logical separation by files
2) there is only one test in the class (do we expect more?)

Reply:
1. Yes, but this file, for instance, has 11 different classes. :D 
So, we could simply group the same class test functions together and keep them at top level, but we might benefit from scoping then under a test class.

2. There were no unit tests for this class at all. I believe the coverage is coming from the api tests. It might be good idea to add then.